src/java.base/share/classes/jdk/internal/math/FloatingDecimal.java

```diff
@@ -1824,6 +1837,17 @@ private static BinaryToASCIIConverter getBinaryToASCIIConverter(float f) {
         return buf;
     }
 
+    static ASCIIToBinaryConverter readDoubleDigits(int decExp, char[] digits, int length) {
+        if (decExp < MIN_DECIMAL_EXPONENT) {
+            return buildZero(BINARY_64_IX, 1);
+        }
+        byte[] buf = new byte[length];
+        for (int i = 0; i < length; i++) {
+            buf[i] = (byte) digits[i];
+        }
+        return new ASCIIToBinaryBuffer(false, decExp, buf, length);
```

Comment:
Since `negSign` is always false, adding a _positive_ somewhere to the method name might be helpful/explicit. Although you could argue that the array parameter itself is telling enough, but one could misinterpret the char array as including a sign character.

Reply:
Thanks, I think "signless" is better.